pkg/agent/controller/networkpolicy/networkpolicy_controller_test.go

```diff
@@ -40,27 +41,47 @@ func newTestController() (*Controller, *fake.Clientset, *mockReconciler) {
 	return controller, clientset, reconciler
 }
 
+// mockReconciler implements Reconciler. It simply records the latest states of rules
+// it have been asked to reconcile, and provide two channels to receive its notification
```

Comment:
```suggestion
// it has been asked to reconcile, and provides two channels to receive its notifications
```

Reply:
Thanks @antoninbas , applied the suggestion